src/harness/unittests/extractMethods.ts

```diff
@@ -1,823 +0,0 @@
-/// <reference path="..\harness.ts" />

```

Comment:
This wasn't detected as a rename because large chunks of it were pulled out into extractRanges.ts and extractSymbolTestHelpers.ts.